manon/breadcrumb-bar.scss

```diff
@@ -57,8 +57,10 @@
         @include icon.icon-format("breadcrumb-bar-");
         @include icon.icon-content("breadcrumb-bar-");
 
-        margin-left: var(--breadcrumb-bar-icon-margin-left);
-        margin-right: var(--breadcrumb-bar-icon-margin-right);
+        & {
+          margin-left: var(--breadcrumb-bar-icon-margin-left);
+          margin-right: var(--breadcrumb-bar-icon-margin-right);
+        }
```

Comment:
It is not needed to use this syntax here right?

```scss
& {

}
```

It would be the same if the lines are placed before the includes.
For example:

```scss
margin-left: var(--breadcrumb-bar-icon-margin-left);
margin-right: var(--breadcrumb-bar-icon-margin-right);

@include icon.icon-format("breadcrumb-bar-");
@include icon.icon-content("breadcrumb-bar-");
```

Maybe I am not right but just checking.

Reply:
The `& { }` block is needed [for forwards compatibility](https://sass-lang.com/documentation/breaking-changes/mixed-decls/) around any regular declarations that follow nested rules, which happens often after `@include` rules because those often themselves contain both regular declarations and nested rules.

In this case I think you're right; the `icon-format` and `icon-content` mixins both lack nested rules. This PR wraps **any** declaration that follows an `@include` rule, just to be safe.

Is the reason you brought this up the [linter complaining about duplicate selectors](https://github.com/minvws/nl-rdo-manon/actions/runs/10456232764/job/28953065655)? It would be great if we could rely on the linter to catch cases where the `& { }` block is not necessary. Unfortunately, the `no-duplicate-selectors` rule is tripped for **all** uses of the `& { }` block. In some cases we could refactor to make wrapping unnecessary, but unfortunately due to the nature of mixins that's not possible everywhere. Which is why we [disabled the lint rule](https://github.com/minvws/nl-rdo-manon/pull/628).

For now that seems the most reasonable approach. If it causes issues in the future we can reconsider our approach.